src/main.py

```diff
@@ -1,22 +1,37 @@
 import logging
-import time
 
 import initialization  # noqa: F401 keep this first, to ensure we're set up for other imports
+from android_utils import share_by_intent
 from android_utils import start_service
 from jnius import autoclass
 from kolibri.main import enable_plugin
 from kolibri.plugins.app.utils import interface
 from kolibri.utils.cli import initialize
-from kolibri.utils.server import _read_pid_file
-from kolibri.utils.server import PID_FILE
-from kolibri.utils.server import STATUS_RUNNING
-from kolibri.utils.server import wait_for_status
+from kolibri.utils.server import BaseKolibriProcessBus
```

Comment:
This will fail awkwardly if we're using Kolibri < 0.16. Should we add an extra check to the Makefile so this gets caught early?

Reply:
Can do - we've mostly relied on tagging and versioning of the installer repos to keep that in check, but doesn't hurt to add extra.

At the very least, we can update the README.